django_mongodb/fields.py

```diff
@@ -32,3 +49,130 @@ def to_python(self, value):
                 code="invalid",
                 params={"value": value},
             ) from None
+
+
+_from_db_value = JSONField.from_db_value
+
+
+def from_db_value(self, value, expression, connection):
+    return (
+        value
+        if isinstance(connection, DatabaseWrapper)
```

Comment:
Any reason to prefer `isinstance(connection, DatabaseWrapper)` over `connection.vendor`? Please add a comment explaining why the method is necessary.